fontforgeexe/charview.c

```diff
@@ -272,7 +272,7 @@ static struct resed charview2_re[] = {
 };
 
 /* return 1 if anything changed */
-static int update_spacebar_hand_tool(CharView *cv) {
+static void update_spacebar_hand_tool(CharView *cv) {
```

Comment:
:shrug: I'm pulling in the [Cygwin patch](http://www.geocities.jp/meir000/fontforge/). I know it correctly implements the behaviour that #6 didn't correctly re-implement (as mentioned earlier), so I guess this information isn't needed.